content/documentation/figure-images.md

```diff
@@ -156,6 +156,25 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 {{< /q-class >}}
 
+## Add Tables
+
+Tables can be embedded in your publication as HTML files. If your table exists in a Word document, resave the Word document as an .html file. In the `static/image/` directory of your project create a new folder called `tables`. Place the .html files in that folder.
```

Comment:
Users shouldn't put tables in directly from Word. And I realized in reading this that that's what you did with Cultural Heritage. The two tables in that book were simple enough so as an example I just did them while I was also fixing some of the style issues you say, here's what the second one looks like now:

```html
<table>
  <thead>
    <tr>
      <th>Denomination</th>
      <th>Mosul-Old City</th>
      <th>Nineveh Plain</th>
      <th>Total</th>
    </tr>
  </thead>
  <tbody>
    <tr><td>Sunni</td><td>35</td><td>6</td><td>41</td></tr>
    <tr><td>Shi’ite</td><td>1</td><td>73</td><td>74</td></tr>
    <tr><td>Yazidi</td><td>0</td><td>26</td><td>26</td></tr>
    <tr><td>Christian</td><td>3</td><td>6</td><td>9</td></tr>
    <tr><td>Other/Misc.</td><td>2</td><td>1</td><td>3</td></tr>
    <tr><td>TOTAL</td><td>41</td><td>114</td><td>155</td></tr>
  </tbody>
</table>
```

That's all that's in the `static/img/tables/table-9-2-stein.html` file. No other HTML tags, no styles, not `id`s or other attributes within the HTML. Just a plain `<table>`.

I think the instruction in the Quire docs here should be that they can use Word as a starting point, but might find hand building tables easier? There are probably also tools to help? Maybe Excel tables would be easier to convert? I'm not sure.

Reply:
I think examples that people can maybe cut and paste from or that explain the html table structure would be good. 

I also found this resource which is really helpful: https://www.w3schools.com/html/html_tables.asp. Might be nice to include, although I noticed you used slightly different html including <thead> and <tbody>. Is that specific to Quire or just a different way of doing it?

Reply:
@geealbers Just pushed a new commit where I added an html table example. What do you think?